pkg/resources/snowflake_credential.go

```diff
@@ -102,14 +102,17 @@ func resourceSnowflakeCredentialCreate(ctx context.Context, d *schema.ResourceDa
 	isActive := d.Get("is_active").(bool)
 	projectId := d.Get("project_id").(int)
 	authType := d.Get("auth_type").(string)
+	database := d.Get("database").(string)
```

Comment:
these need to be added to the resource schema further up in this file too (causing tests to fail) 😄